charts-packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -915,7 +916,7 @@ export abstract class Chart extends Observable implements AgChartInstance {
             if (!match || distance == null) {
                 continue;
             }
-            if (!result || result.distance > distance) {
+            if ((!result || result.distance > distance) && distance <= (maxDistance || Infinity)) {
```

Comment:
Just wondering if Falsey evaluation might be a problem, should we use the `??` operator instead here:
```suggestion
            if ((!result || result.distance > distance) && distance <= (maxDistance ?? Infinity)) {
```